content/news/2023-05-06-ABPHM/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Galaxy taking part @ Applied Bioinformatics & Public Health Microbiology 2023
+date: '2023-05-06'
+tease: "From the 3rd to the 5th of May, Engy Nasr from the Freiburg Galaxy team travelled to Hinxton, England to join and present her work at the Applied Bioinformatics & Public Health Microbiology 2023 conference"
+location:
+  name: Hinxton, UK
+tags: [conference, talk, bioinformatics showcase, pathogen detection]
+authors: engynasr
+supporters:
+- microbial genomics
```

Comment:
@bebatut what supportes should we add here? Any idea?

Reply:
EOSC-Life for sure. Maybe ELIXIR too